src/Controller/OG.php

```diff
@@ -231,4 +232,83 @@ public static function MembershipStorage() {
   public static function MembershipDefault() {
     return array('type' => 'og_membership_type_default');
   }
+
+  /**
+   * Get the groups an entity is associated with.
+   *
+   * @param $entity_type
+   *   The entity type. Defaults to 'user'
+   * @param $entity
+   *   (optional) The entity object or entity ID. If empty, and $entity_type is
+   *   "user", the current user will be used.
+   * @param $states
+   *   (optional) Array with the state to return. Defaults to active.
+   * @param $field_name
+   *   (optional) The field name associated with the group.
+   *
+   * @return array
+   *  An array with the group's entity type as the key, and array - keyed by
+   *  the OG membership ID and the group ID as the value. If nothing found,
+   *  then an empty array.
+   */
+  public static function getEntityGroups($entity_type = 'user', $entity = NULL, $states = array(OG_STATE_ACTIVE), $field_name = NULL) {
```

Comment:
can we change $entity to `$entity_id` - so it gets only an ID, not the whole object?